docs/openapi.yaml

```diff
@@ -78,157 +78,163 @@ paths:
       parameters:
       - in: query
         name: category
-        description: "Term to search for in resource."
+        description: "Result category code, category codes are defined in the Cristin API documentation."
```

Comment:
We shouldent redirect to a documentation that will be taken down when NVA replaced Cristin